### ISSUE TITLE
Message banner colors lack contrast

### ISSUE THREAD

**Issue by WesleyBranton** [WesleyBranton/New-Browser-Style#15]
The color contrast of the message banners is insufficient to pass the 3:1 contrast ratio requirement of 1.4.11 in WCAG AA.
